app/state.go

```diff
@@ -2075,6 +2075,12 @@ func (c *controlMapPauseState) Handle(ctx *StateContext, controller Controller)
 
 	updateMapFromServerIn := time.Until(nextServerMapRefresh)
 
+	// For rapid feedback, use the smallest number of the map refresh time
+	// and the update poll interval. The latter is often much shorter.
+	if updateMapFromServerIn > controller.GetUpdatePollInterval() {
```

Comment:
I think we have a Min function somewhere? :thinking:

Reply:
Yes, but for ints. Golang is really picky about types, so here it made more sense to just a do a check.